code/dynamic_programming/src/factorial/factorial.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Factorial do
+  def factorial(0), do: 1
+  def factorial(1), do: 1
+  def factorial(num) do
+    num * factorial(num - 1)
+  end
+end
```

Comment:
There should be a newline at the end of the file as is traditional for Elixir.  Generally running the code through the built-in Elixir Formatter will format things in the 'proper' way.

Reply:
Elixir has a formatter?